spec/rubocop/cop/bundler/duplicated_gem_spec.rb

```diff
@@ -85,5 +85,21 @@
         expect(cop.highlights).to eq(["gem 'rubocop', path: '/path/to/gem'"])
       end
     end
+
+    context 'and same gems have different version specifications' do
+      let(:source) { <<-END }
+        if RUBY_VERSION < '2.2.5'
+          gem 'beaker', '~> 2.0', require: false
+          gem 'beaker-rspec', '~> 5.0', require: false
+        else
+          gem 'beaker-rspec', require: false
+        end
+      END
```

Comment:
Does it register an offense if I don't have a conditional?
eg.
```
gem 'beaker-rspec'
gem 'beaker-rspec', '~> 5.0'
```

Also please check https://github.com/bbatsov/rubocop/issues/3752#issuecomment-267816609